Supporting .net standard 1.6.

iremmats opened the pull request:
New major version. 
.NET standard 2.0.
People have to upgrade their Visual Studio... :)

marbergq commented:
Does not build, i don't think the FormatterServices are included in the netstandard, thus needs an nuget dependency (?)

iremmats commented:
"It works on my machine..." :)

Think it's AppVeyor that doesn't support it yet.

See https://andrewlock.net/building-asp-net-core-2-0-preview-2-packages-on-appveyor/. Might not be 100% relevant.

marbergq commented:
FormatterServices.GetUninitializedObject in netstandard2.0 is a wrapper for          RuntimeHelpers.GetUninitializedObject(type); 

https://github.com/dotnet/corefx/blob/6c8411036d4b196c48433ac5e0118b2de525c01c/src/System.Runtime.Serialization.Formatters/src/System/Runtime/Serialization/FormatterServices.cs#L185 

Does that exist in 1.6?

iremmats commented:
Cleaned two packages that weren't used and changed back to .net standards 1.6 AND net46. Is it OK to not support 4.5?
The line that caused all the "problems". In which scenario is it useful?
https://github.com/collector-bank/common-restcontracts/blob/94072281aa9482bff62453e2e69c74008c015d2d/src/Collector.Common.RestContracts/RequestBase.cs#L45

marbergq commented on src/Collector.Common.RestContracts/RuntimeHelper.cs:
```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
```
unnecessary using

marbergq commented on src/Collector.Common.RestContracts/RuntimeHelper.cs:
```diff
@@ -0,0 +1,30 @@
+
+    public class RuntimeHelper
+    {
+        private static readonly Func<Type, object> getUninitializedObjectDelegate =
```
check indentation
